test-packages/macro-tests/vendor/prepend/four.js

```diff
@@ -1 +1 @@
-self.ORDER.push('prepend/four');
+self.ORDER && self.ORDER.push('prepend/four');
```

Comment:
This guard seems to be masking another issue, please ensure the root cause is addressed.

Reply:
My existing PR #322 addresses this issue: https://github.com/embroider-build/embroider/pull/322

Specifically, this commit: https://github.com/embroider-build/embroider/pull/322/commits/382d9e5b32748c7a5d6666f1e73596cc8d7158b0

Reply:
removed changes and fixed this issue as part o theis PR - https://github.com/embroider-build/embroider/pull/336